docroot/sites/clas.uiowa.edu/modules/clas_core/clas_core.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Install tasks.
+ */
+
+use Drupal\redirect\Entity\Redirect;
+
+/**
+ * CLAS redirects.
+ */
+function clas_core_update_8001() {
```

Comment:
```suggestion
function clas_core_update_10001() {
```
we usually align with the version of drupal we are on at the time these are triggered